src/main/drivers/vtx_common.h

```diff
@@ -32,6 +32,36 @@ typedef enum {
     VTXDEV_UNKNOWN    = 0xFF,
 } vtxDevType_e;
 
+
+// VTX magic numbers
+
+#define VTX_COMMON_BAND_USER      0
+#define VTX_COMMON_BAND_A         1
+#define VTX_COMMON_BAND_B         2
+#define VTX_COMMON_BAND_E         3
+#define VTX_COMMON_BAND_FS        4
+#define VTX_COMMON_BAND_RACE      5
+
+// RTC6705 RF Power index "---", 25 or 200 mW
+#define VTX_6703_POWER_OFF     0
```

Comment:
@AndersHoglund: Thanks for moving those. As cumbersome as it is that all the different hardware protocols define their own power levels, at least having the defines for them in a shared place means everybody can make use of the same defines.

One question I have: Your code is using both '6705' and '6703'. Are they different models of the same chip, and if so would it make sense to standardise the naming to one of the two?

Reply:
6703 is a typo. Thanks, good find.

Reply:
I was wondering about that, but then Google found a number of results to do with video transmitting when searching for rtc6703, so I wasn't sure. :grinning: